src/vscode-dts/vscode.proposed.interactive.d.ts

```diff
@@ -82,7 +83,7 @@ declare module 'vscode' {
 		provideInteractiveEditorResponse(session: S, request: InteractiveEditorRequest, progress: Progress<InteractiveEditorProgressItem>, token: CancellationToken): ProviderResult<R>;
 
 		// eslint-disable-next-line local/vscode-dts-provider-naming
-		handleInteractiveEditorResponseFeedback?(session: S, response: R, kind: InteractiveEditorResponseFeedbackKind): void;
+		handleInteractiveEditorResponseFeedback?(session: S, response: R, kind: InteractiveEditorResponseFeedbackKind, shouldReportIssue?: boolean): void;
```

Comment:
This is a bit weird, I'd very much prefer to extend `InteractiveEditorResponseFeedbackKind` with another value like `Bug`